packages/annotation-ui/src/panels/ContextPanel.js

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from "react";
+import { Context } from "../model/Store";
+import { Slider } from "@blueprintjs/core";
+
+export default function ContextPanel() {
+  const { set, get, invoke } = useContext(Context);
+
+  const videoLayerData = get(["layers", "Video", "data"]);
+  const videoLoaded = !!videoLayerData;
+
+  return videoLoaded ? (
+    <div>
+      Current video time (seconds): {videoLayerData.playedSeconds}
+      <div style={{ padding: 20 }}>
+        <Slider
```

Comment:
Could this be defined in the layer and rendered here? I'm imagining a scenario where each layer could should additional info such as tracks, ticks, markers etc so that it any interesting sections of the video could be easily discovered via the UI.

Reply:
Ah yes, I agree. I'll add a TODO comment in the code here, will update to extract this out in the future.

I can imagine this ContextPanel serving two purposes: it could be either provide context at a Task level or at a Layer level (or... we just leave it up to the task designer an have them decide - so I guess that's a 3rd option).

Regardless, I'm trying to condense as much of the custom code into one file, currently `Task.js`, so I'd like to move this code there.